ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/PoWSolver.java

```diff
@@ -26,11 +26,15 @@
 
 import com.google.common.base.Stopwatch;
 import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.concurrent.ExpiringMap;
 import org.apache.tuweni.units.bigints.UInt256;
 
 public class PoWSolver {
 
+  private static final int MAX_OMMER_DEPTH = 8;
```

Comment:
For another PR, another `--X` candidate.

Reply:
https://github.com/hyperledger/besu/pull/2584 is open